### PR TITLE
Loot update

### PR DESCRIPTION
**Pull request by Uberbagel**

### CODE REVIEW

File: scripts/items/loot/rf_geist_tear_item.nut [hunk 1]
```diff
@@ -0,0 +1,77 @@
+this.rf_geist_tear_item <- this.inherit("scripts/items/item", {
+	m = {},
+	function create()
+	{
+		this.item.create();
+		this.m.ID = "misc.rf_geist_tear";
+		this.m.Name = "Geist Tear";
+		this.m.Description = "When a geist leaves this plane, sometimes a gem is left behind. Wise men believe it is the ethereal essence of the spirit made manifest.";
+		this.m.Icon = "loot/rf_geist_tear.png";
+		this.m.SlotType = this.Const.ItemSlot.None;
```
Comment (Darxo) [inline scripts/items/loot/rf_geist_tear_item.nut:10]: lots of `this.Const` uses etc. in this file. 
Auto replacing them just within this file is quickly done

**Review by Darxo — CHANGES_REQUESTED** [review]

File: scripts/items/loot/rf_geist_tear_item.nut [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+		}
+		else
+		{
+			return ::Math.floor(this.getValue());
+		}
+	}
+
+	function playInventorySound( _eventType )
+	{
+		this.Sound.play("sounds/combat/armor_leather_impact_03.wav", ::Const.Sound.Volume.Inventory);
```
Comment (LordMidas) [inline scripts/items/loot/rf_geist_tear_item.nut:74]: I think the glass bottle sound would be better i.e. the one for holy water bottle or potions etc. Or some other sound that goes more in line with it being a gem/glass? Not a huge deal though.

File: scripts/items/loot/rf_geist_tear_item.nut [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+		this.m.Icon = "loot/rf_geist_tear.png";
+		this.m.SlotType = ::Const.ItemSlot.None;
+		this.m.ItemType = ::Const.Items.ItemType.Misc | ::Const.Items.ItemType.Loot;
+		this.m.IsDroppedAsLoot = true;
+		this.m.Value = 300;
+	}
+
+	function getTooltip()
+	{
+		local result = [
```
Comment (LordMidas) [inline scripts/items/loot/rf_geist_tear_item.nut:18]: Cleaner code is:
```squirrel
return [
	{
		id = 1,
		type = "title",
		text = this.getName()
	},
	{
		id = 2,
		type = "description",
		text = this.getDescription()
	},
	{
		id = 66,
		type = "text",
		text = this.getValueString()
	}
];

**Review by LordMidas — APPROVED** [review]
Consider approved after the changes suggested have been implemented.

File: scripts/items/loot/rf_geist_tear_item.nut (hunk 2 above)
Comment (LordMidas) [inline scripts/items/loot/rf_geist_tear_item.nut:74]: `::Sound`

File: mod_reforged/hooks/entity/tactical/enemies/ghost.nut [hunk 4]
```diff
@@ -4,7 +4,7 @@
 		this.actor.onInit();
 		this.setRenderCallbackEnabled(true);
 		local b = this.m.BaseProperties;
-		b.setValues(this.Const.Tactical.Actor.Ghost);
+		b.setValues(::Const.Tactical.Actor.Ghost);
```
Comment (LordMidas) [inline mod_reforged/hooks/entity/tactical/enemies/ghost.nut:7]: Don't do these specific `this.Const` to `::Const` changes in this commit (i.e. lines 7, 27, 28). You can discard these ones completely, we'll do them in a later cleanup commit in the codebase. Only fix the `::Const` in the new code you added.

File: mod_reforged/hooks/entity/tactical/enemies/direwolf.nut [hunk 5]
```diff
@@ -61,4 +61,33 @@
 			this.m.Skills.add(this.new("scripts/skills/perks/perk_rf_survival_instinct"));
 		}
 	}
+
```
Comment (LordMidas) [inline mod_reforged/hooks/entity/tactical/enemies/direwolf.nut:64]: The commit message is too long. Please change it to the following i.e. shorter message and move the details to a separate paragraph with a line gap.
```
refactor: adjust loot drops of certain entities

Specifically direwolf, ghoul, hexe, hyena, necromancer, schrat, serpent, spider, unhold, vampire and goblins.
```

File: scripts/items/loot/rf_geist_tear_item.nut (hunk 2 above)
Comment (Uberbagel) [reply scripts/items/loot/rf_geist_tear_item.nut:74]: I used the same as the Soul Splinter, which is where the recolored icon comes from. I would like to leave it as is for now.

**Review by Darxo — APPROVED** [review]

**Pull request merged**